exercise-src/react-native/11-navigation/03-problem/src/components/RestaurantHeader/RestaurantHeader.test.tsx

```diff
@@ -3,13 +3,28 @@ import { render, screen } from "@testing-library/react-native"
 import { restaurantWithAddress, restaurantWithoutAddress } from "./mocks"
 import RestaurantHeader from "./RestaurantHeader"
 
-describe("RestaurantHeader component", () => {
-  it("renders the restaurant name", () => {
+describe("Components/RestaurantHeader", () => {
+  it("renders", () => {
     render(<RestaurantHeader restaurant={restaurantWithAddress} />)
+    expect(screen.getByText(/\$\$\$/i)).toBeOnTheScreen()
+    expect(screen.getByText("$$$", { exact: false })).toBeOnTheScreen()
+    expect(
+      screen.getByText("Hours: M-F 10am-11pm", { exact: false }),
+    ).toBeOnTheScreen()
+
     expect(screen.getByText(/Test Restaurant/)).toBeOnTheScreen()
+    expect(screen.getByText(/\$\$\$/i)).toBeOnTheScreen()
+    expect(screen.getByText("$$$", { exact: false })).toBeOnTheScreen()
+    expect(
+      screen.getByText("Hours: M-F 10am-11pm", { exact: false }),
+    ).toBeOnTheScreen()
   })
 
-  it("renders the restaurant address when provided", () => {
+  it("does without an address", () => {
```

Comment:
Phrasing is a little weird here.

Reply:
I fixed this locally but didnt' commit cause I want to get this merged so I can be done. =] it should be "renders" not "does"

Reply:
Yeah, I would definitely merge this in as-is.